pom.xml

```diff
@@ -13,18 +13,15 @@
         <project.build.resourceEncoding>UTF-8</project.build.resourceEncoding>
         <maven.compile.encoding>UTF-8</maven.compile.encoding>
 
-        <!-- require log4j 2.6, until 2.8 released with fix for
-             https://issues.apache.org/jira/browse/LOG4J2-1642 -->
-        <log4j.version>2.8.2</log4j.version>
-        <powermock.version>1.7.0</powermock.version>
-
         <test.integration.pattern>**/*IntegrationTest*.java</test.integration.pattern>
+
+        <powermock.version>1.7.3</powermock.version>
     </properties>
 
     <parent>
         <groupId>org.sonatype.oss</groupId>
         <artifactId>oss-parent</artifactId>
-        <version>7</version>
+        <version>9</version>
```

Comment:
what impact it introduces?

Reply:
Ensures the latest recommended maven central plugins are up to date.